src/types/enums.ts

```diff
@@ -0,0 +1,14 @@
+export enum FilterTypes {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
+
+export enum ErrorTypes {
+  def = '',
+  loadErr = 'Unable to load todos',
+  titleErr = 'Title should not be empty',
+  addErr = 'Unable to add a todo',
+  delErr = 'Unable to delete a todo',
+  updErr = 'Unable to update a todo',
```

Comment:
Make up better names. Also, they should start with an upper letter, like in `enum` above